book/07-error-handling.html

```diff
@@ -181,9 +181,9 @@ <h2>Error and Or_error</h2>
 
           <p>
 	    S-expressions are supported by the Sexplib package that
-            is distributed with Core and is the most common
-            serialization format used in Core. Indeed, most types in
-            Core come with built-in s-expression converters.
+            is distributed with Base and is the most common
```

Comment:
forward reference to Part II and the serialization chapter?

Reply:
There's already a forward reference a few lines down.